manifests/charts/README-helm3.md

```diff
@@ -1,57 +1,64 @@
-# Helm3 support
+# Helm v3 support
 
 ## Install
 
-The manifests/ templates support both helm2 and helm3. Please do not introduce helm3-specific changes, many
-users are still using helm2 and the operator is currently using the helm2 code to generate.
+The Helm charts are support both by Helm v2 and Helm v3. Please do not introduce Helm v3 specific changes as many
+users are still using Helm v2 and the operator is currently using the Helm v2 code to generate.
 
-To install in helm3 you must first create a namespace that you wish to install the following charts to:
+To install with Helm v3, you must first create the namespace that you wish to install in if the namespace does not exist already. The default namespace used is `istio-system` and can be created as follows:
 
-```shell script
- kubectl create namespace istio-system
+```console
+kubectl create namespace istio-system
 ```
 
-We have few charts:
+The charts are as follows:
 
-- 'base' creates cluster-wide CRDs, cluster bindings, cluster resources and the istio-system namespace.
-  It is possible to customize the namespace, but not recommended.
+- 'base' creates cluster-wide CRDs, cluster bindings and cluster resources. It is possible to change the namespace from `istio-system` but it is not recommended.
 
-```shell script
- helm3 install istio-base -n istio-system manifests/charts/base
+```console
+helm install istio-base -n istio-system manifests/charts/base
 ```
 
 - 'istio-control/istio-discovery' installs a revision of istiod.  You can install it multiple times, with different revisions.
-TODO: remove the need to pass -n istio-system
 
-```shell script
- helm3 install -n istio-system istio-17 manifests/charts/istio-control/istio-discovery
+```console
+ helm install -n istio-system istio-17 manifests/charts/istio-control/istio-discovery -f manifests/charts/global.yaml
```

Comment:
I thought we removed the requirement for this. @jacob-delgado is it still required, and if not can it be removed?

Reply:
It should be removed after https://github.com/istio/istio/pull/25186

Reply:
That is a typo by me, apologies. It is working without the global values.

Reply:
Removed.